### PR TITLE
Universal Print - Fixing inconsistencies between docs & schema for Beta and v1.0 APIs

### PR DESCRIPTION
**Pull request by braedenp-msft**

### CODE REVIEW

File: api-reference/v1.0/resources/printjobconfiguration.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ A group of settings that a printer should use to print a job.
 |pagesPerSheet|Int32|The number of document pages to print on each sheet.
 |multipageLayout|[printMultipageLayout](enums.md#printmultipagelayout-values)|The direction to lay out pages when multiple pages are being printed per sheet. Valid values are described in the following table.|
 |collate|Boolean|Whether the printer should collate pages wehen printing multiple copies of a multi-page document.|
-|scaling|[printScaling](enums.md#printscaling-values)|Specifies how the printer should scale the document data to fit the requested media. Valid values are described in the following table.|
+|scaling|[printScaling](enums.md#printscaling-values)|Specifies how the printer should scale the document data to fit the requested media. Valid values are described in the following table.|]
```
Comment (nilakhan) [inline api-reference/v1.0/resources/printjobconfiguration.md:36]: There's an extra ']' at the end of line 36

Comment (braedenp-msft) [reply api-reference/v1.0/resources/printjobconfiguration.md]: Fixed!

File: api-reference/beta/resources/print.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ When accompanied by a Universal Print subscription, the Print feature enables ma
 |printers|[printer](printer.md) collection|The list of printers registered in the tenant.|
 |shares|[printerShare](printershare.md) collection|The list of printer shares registered in the tenant.|
 |connectors|[printConnector](printconnector.md) collection|The list of available print connectors.|
+|operations|[printOperation](../resources/printoperation.md) collection|The list of print long running operations.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/print.md:45]: Shouldn't this navigation property have a List operations method? It is a collection. Please clarify.

File: api-reference/beta/resources/print.md [hunk 3]
```diff
@@ -42,6 +42,8 @@ When accompanied by a Universal Print subscription, the Print feature enables ma
 |printers|[printer](printer.md) collection|The list of printers registered in the tenant.|
 |shares|[printerShare](printershare.md) collection|The list of printer shares registered in the tenant.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/print.md:43]: Missing in this table: 
- reports navigation property

Comment (isvargasmsft) [inline api-reference/beta/resources/print.md:43]: Global: Do all these changes have changelog entries already? If not, please create changelog entries for all.

File: api-reference/beta/resources/printconnector.md [hunk 4]
```diff
@@ -35,7 +35,6 @@ Represents a print connector that has been registered by using a Universal Print
 |deviceHealth|[deviceHealth](devicehealth.md)|The connector's device health.|
 |location|[printerLocation](printerlocation.md)|The physical and/or organizational location of the connector.|
 |registeredDateTime|DateTimeOffset|The DateTimeOffset when the connector was registered.|
-|registeredBy|[userIdentity](useridentity.md)|The user who registered the connector.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/printconnector.md:38]: Please remove this one from the JSON representation section also.

File: api-reference/beta/resources/printercapabilities.md [hunk 5]
```diff
@@ -25,7 +25,6 @@ Represents the capabilities reported by a printer.
 |qualities|printQuality collection|The print qualities supported by the printer.|
 |dpis|Int32 collection|The list of print resolutions in DPI that are supported by the printer.|
 |duplexModes|printDuplexMode collection|The list of duplex modes that are supported by the printer. Valid values are described in the following table.|
-|queueBufferSizeInBytes|Int32|The maximum print job queue size that can be stored by the printer.|
 |copiesPerJob|[integerRange](integerrange.md)|The range of copies per job supported by the printer.|
 |finishings|printFinishing collection|Finishing processes the printer supports for a printed document.|
 |mediaColors|String collection|The media (i.e., paper) colors supported by the printer.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/printercapabilities.md:30]: Missing in this table: 
image


Please also add these properties to the JSON representation.

File: api-reference/beta/resources/printerlocation.md [hunk 6]
```diff
@@ -22,7 +22,7 @@ Represents the physical and hierarchical location of a printer.
 |longitude|Double|The longitude that the printer is located at.|
 |altitudeInMeters|Int32|The altitude, in meters, that the printer is located at.|
 |streetAddress|String|The street address where the printer is located.|
-|subUnit|String collection|The subunit hierarchy where the printer is located. The elements should be in hierarchical order. For example, if a campus is divided into different sections, the hierarchy might look like this: `["East Wing", "Block A"]`|
+|subunit|String collection|The subunit hierarchy where the printer is located. The elements should be in hierarchical order. For example, if a campus is divided into different sections, the hierarchy might look like this: `["East Wing", "Block A"]`|
 |city|String|The city that the printer is located in.|
 |postalCode|String|The postal code that the printer is located in.|
 |countryOrRegion|String|The country or region that the printer is located in.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/printerlocation.md:28]: missing in this table:
- floorNumber
- roomNumber

Comment (isvargasmsft) [reply api-reference/beta/resources/printerlocation.md:28]: please apply these changes to the JSON representation section.

File: api-reference/beta/resources/printjob.md [hunk 7]
```diff
@@ -37,11 +37,11 @@ Represents a print job that has been queued for a printer.
 |isFetchable|Edm.Boolean|If true, document can be fetched by printer.|
 |redirectedFrom|Edm.String|Contains the source job URL, if the job has been redirected from another printer.|
 |redirectedTo|Edm.String|Contains the destination job URL, if the job has been redirected to another printer.|
+|createdBy|[userIdentity](useridentity.md)| Read-only. Nullable.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/printjob.md:40]: please update the JSON representation.

File: api-reference/beta/resources/printjobconfiguration.md [hunk 8]
```diff
@@ -36,6 +36,7 @@ A group of settings that a printer should use to print a job.
 |multipageLayout|printMultipageLayout|The direction to lay out pages when multiple pages are being printed per sheet. Valid values are described in the following table.|
 |collate|Boolean|Whether the printer should collate pages wehen printing multiple copies of a multi-page document.|
 |scaling|printScaling|Specifies how the printer should scale the document data to fit the requested media. Valid values are described in the following table.|
+|fitPdfToPage|Boolean|If true, PDF pages are scaled to match the print media size.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/printjobconfiguration.md:39]: Global: please also update all JSON representations in this PR.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Thank you very much for updating these docs. 

Please take the time to also update the JSON representation sections and review a few missing properties. 

Thank you very much!

Isaac Vargas

**Comment by isvargasmsft** [comment]
Hello @braedenp-msft, 

What is the status of this PR? Please let me know if you have any questions regarding the comments above. 

Once you have addressed the feedback, please add the label "review feedback addressed". 

Kind regards, 

Isaac

File: api-reference/beta/resources/print.md [hunk 9]
```diff
@@ -42,6 +42,8 @@ When accompanied by a Universal Print subscription, the Print feature enables ma
 |printers|[printer](printer.md) collection|The list of printers registered in the tenant.|
```
[the remark this answers is not in the document]
Comment (braedenp-msft) [reply api-reference/beta/resources/print.md:42]: Please see the `shares` property below.

File: api-reference/beta/resources/print.md (hunk 3 above)
Comment (braedenp-msft) [reply api-reference/beta/resources/print.md:43]: Our reporting endpoints are under reportRoot (outside of the print entity).

File: api-reference/beta/resources/print.md (hunk 2 above)
Comment (braedenp-msft) [reply api-reference/beta/resources/print.md:45]: I think we need to publish a new page. @nilakhan who's the best person to help with content for a page documenting how to list operations?

File: api-reference/beta/resources/print.md (hunk 3 above)
Comment (braedenp-msft) [reply api-reference/beta/resources/print.md:43]: Yep. We just didn't update the Beta docs.

File: api-reference/beta/resources/printserviceendpoint.md [hunk 10]
```diff
@@ -25,7 +25,7 @@ Represents URI and identifying information for a print service instance.
 | Property     | Type        | Description |
 |:-------------|:------------|:------------|
 |displayName|String|A human-readable display name for the endpoint.|
-|name|String|A unique name that identifies the service that the endpoint provides. Possible values are: `discovery` (Discovery Service), `notification` (Notification Service), `ipp` (IPP Service), and `registration` (Registration Service). Read-only.|
+|id|String|A unique name that identifies the service that the endpoint provides. Possible values are: `discovery` (Discovery Service), `notification` (Notification Service), `ipp` (IPP Service), and `registration` (Registration Service). Read-only.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/printserviceendpoint.md:28]: please update the JSON representation section on this topic. In addition, please clarify if this is the ID inherited from entity or an enum type. 

If this is the id inherited from entity, then please update the description of the property as well.

**Comment by braedenp-msft** [comment]
These issues still need to be fixed. Haven't had time to do it yet. Commenting to keep the PR open.

**Pull request closed without merging**